src/presence.js

```diff
@@ -0,0 +1,35 @@
+import * as R from 'ramda';
+
+/**
+ * Check if value is empty or undefined then returns null, otherwise, returns itself.
+ *
+ * @func presence
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.29.0|v2.29.0}
+ * @category Object
+ * @sig k -> null | k
+ * @param {k} val The value to test
+ * @return {null | k}
+ * @example
+ *
+ * presence({ foo: 'foo' }) // { foo: 'foo' }
```

Comment:
```suggestion
 * RA.presence({ foo: 'foo' }) // => { foo: 'foo' }
```

Please prefix all the examples with `RA.` prefix. Please also notice that we have a convention of showing the expected result using `// => {val}` convention.